migrations/2_deploy_dharma_contracts.js

```diff
@@ -1,22 +1,44 @@
+const OWNERS = require("./OWNERS");
+
 module.exports = (deployer, network, accounts) => {
+    // Import the Dharma contracts.
     const PermissionsLib = artifacts.require("PermissionsLib");
-    const DummyContract = artifacts.require("DummyContract");
     const DebtRegistry = artifacts.require("DebtRegistry");
     const DebtToken = artifacts.require("DebtToken");
     const DebtKernel = artifacts.require("DebtKernel");
     const RepaymentRouter = artifacts.require("RepaymentRouter");
     const TokenTransferProxy = artifacts.require("TokenTransferProxy");
     const MultiSigWallet = artifacts.require("MultiSigWallet");
 
+    // We switch on the network to ensure we're configuring our MultiSigWallet
+    // accordingly.
+    let owners;
+    switch (network) {
+        case "live":
+            owners = OWNERS.owners;
+            break;
+        case "kovan":
+        case "development":
+            owners = accounts;
+            break;
+        default:
+            throw "invalid network";
+    }
+
+    const required = Math.floor(owners.length / 2);
```

Comment:
Ideally would like to parametrize this somehow as well -- i.e. have the `threshold` specified in the `OWNERS.js`/`MULTISIG_SIGNATORIES.js` file.

Reply:
Can do!

Reply:
🎸